AEPIdentity/Sources/CustomIdentity.swift

```diff
@@ -24,18 +24,18 @@ public class CustomIdentity: Identifiable, Codable {
     ///   - type: Type of the identifier
     ///   - identifier: The identifier
     ///   - authenticationState: Authentication state for the identifier
-    init(origin: String?, type: String?, identifier: String?, authenticationState: MobileVisitorAuthenticationState) {
+    public init(origin: String?, type: String?, identifier: String?, authenticationState: MobileVisitorAuthenticationState) {
```

Comment:
Why do we need this class to be public? Looking at the Identity public APIs it looks like the sync apis accept free form dictionaries, while the getter returns Identifiable.

Reply:
fixed.

Reply:
can we remove the modifier from the class too? cc @shalehaha

Reply:
removed that too. the original intention was for this class to be public, but that has been replaced with the `Identifiable` protocol.

Reply:
yep, should change it to private